versions/3_12/samples/config.py

```diff
@@ -1,20 +1,22 @@
 import os
 
 class Config:
-    def __init__(self, address, store_id, api_key, terminal_id):
+    def __init__(self, address, store_id, api_key, terminal_id, show_tips):
         self.address = address
         self.store_id = store_id
         self.api_key = api_key
         self.terminal_id = terminal_id
+        self.show_tips = show_tips
         print(self)
 
     def __str__(self):
-        return f'Config: address: {self.address}, store_id = {self.store_id,} api_key = {self.api_key}, terminal_id = {self.terminal_id}'
+        return f'Config: address: {self.address}, store_id = {self.store_id,} api_key = {self.api_key}, terminal_id = {self.terminal_id}, show_tips = {self.show_tips}'
 
 def load_config():
     # load the configuration from the ENV variables
     config = Config(address=os.getenv('KODY_ADDRESS'),
                     store_id=os.getenv('KODY_STORE_ID'),
                     api_key=os.getenv('KODY_API_KEY'),
-                    terminal_id=os.getenv('KODY_TERMINAL_ID'))
+                    terminal_id=os.getenv('KODY_TERMINAL_ID'),
+                    show_tips=os.getenv('SHOW_TIPS'))
```

Comment:
IMO, this should be configured on a per-payment basis...

Reply:
But I guess _as an example_ this is fine

Reply:
We could use 
`tips = input("Show tips?")`
Any input will be true or enter will be false